scss/_modal.scss

```diff
@@ -150,6 +150,8 @@
   overflow: scroll;
 }
 
+.modal-sm { max-width: $modal-sm; }
```

Comment:
Why are you adding this line? it brakes the small modal on xs sizes.

Reply:
My tests
modal-sm on a wide screen:
![normal](https://user-images.githubusercontent.com/2226999/44420242-b6baf180-a57d-11e8-9fae-1e47bfa2efbf.PNG)

modal-sm on a narrow screen, original:
![original](https://user-images.githubusercontent.com/2226999/44420248-b9b5e200-a57d-11e8-8836-9a0fa44f6ccc.PNG)

modal-sm on a narrow screen, patched:
![patched](https://user-images.githubusercontent.com/2226999/44420251-bb7fa580-a57d-11e8-9745-12e8ef7ab4d7.PNG)

Reply:
Yes, I'd say that the result is accurate, we want it full screen on small viewports.

I'd remove that line.

Reply:
ok

Reply:
Ok, line removed.